### PR TITLE
#164369493-implement custom pagination component

### PR DESCRIPTION
**Pull request by ozone4real**
#### What does this PR do?
Create a custom pagination component.

#### How should this be manually tested?
Clone the Repo, run npm install, then run npm test.

#### What are the relevant pivotal tracker stories?
#164369493

#### Screenshots (if appropriate)

Screen Shot 2019-03-12 at 6 55 19 AM

Screen Shot 2019-03-12 at 6 55 38 AM

### CODE REVIEW

File: src/styles/components/_pagination.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.pagination .prev-page {
+  height: 0;
+  width: 0;
+  margin-top: 0.2rem;
+  border-right: 1.7rem solid #0b41cd;
+  border-bottom: 1rem solid transparent;
+  border-top: 1rem solid transparent;
+}
```
Comment (p-funky) [inline src/styles/components/_pagination.scss:35]: This has the same styling with `.pagination next-page`. Consider merging them.

File: src/styles/components/_pagination.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+.prev-page.caret-left-arrow-disabled {
+  border-right-color: #d8d8d8;
+}
+
+.next-page.caret-right-arrow-disabled {
+  border-left-color: #d8d8d8;
+}
```
Comment (p-funky) [inline src/styles/components/_pagination.scss:43]: This has the same styling with `.prev-page.caret-left-arrow-disabled`. Consider merging them.

**Review by bobbylemm — DISMISSED** [review]

File: src/styles/components/_pagination.scss (hunk 1 above)
Comment (ozone4real) [reply src/styles/components/_pagination.scss:35]: Not exactly the same. Only difference is `border-right` and `border-left`. I'll merge the common properties together tho.

File: src/styles/components/_pagination.scss (hunk 2 above)
Comment (ozone4real) [reply src/styles/components/_pagination.scss:43]: One is ` border-left-color`, the other is `border-right-color`.

**Review by biolabalo — DISMISSED** [review]

**Review by EfeAgare — DISMISSED** [review]

**Review by biolabalo — DISMISSED** [review]

**Review by p-funky — APPROVED** [review]

**Review by divinediscipline — APPROVED** [review]
LGTM

**Pull request merged**